website/docs/r/cluster.html.markdown

```diff
@@ -307,11 +307,13 @@ Physical location of the region.
 
 -> **NOTE:** Prior to setting these options please ensure you read https://docs.atlas.mongodb.com/cluster-config/additional-options/.
 
+-> **NOTE:** This argument has been changed to type list make sure you have the proper syntax. The list can have only one  item maximum.
```

Comment:
Just a small edit to make this a bit clearer for when we release: 

This argument has been changed as of version 0.6.0.  The is now a type list, make sure you have the new syntax in your configuration as show below (remove the previously required = sign). Note, the list can have only one item maximum.

Reply:
Also checked the cluster data source and the advanced config items never got added so no work now but will add this task to the back, fyi.

Reply:
When I read `The list can have only one  item maximum` I confused with not being able to do this:

```
advanced_configuration  {
    fail_index_key_too_long            = true
    javascript_enabled                   = true
}
```

But actually this is ok and what is not allowed is this:

```
advanced_configuration  {
    fail_index_key_too_long            = true
}
advanced_configuration  {
    javascript_enabled                   = true
}
```

Is this understanding correct?

If yes, then I suggest maybe a more clear sentence, for example, `You should only define a single advanced_configuration block` (or something like that)...

Reply:
Good point - it's literally the different between having an equal sign before:
advanced_configuration  = { 
and having none now
advanced_configuration  {   

So definitely want to change it if not clear.  I will edit my comment to make it clearer.